--- plain_text.orig	2004-12-24 08:36:11.000000000 +0000
+++ plain_text.changed	2004-12-24 08:45:53.000000000 +0000
@@ -1,8 +1,7 @@
-Following a recent article which corrolated a language's success with
-the facial hair of its designer, Matz offered this spritely pic of his
-Christmas beard. If your resolution is too high to make out the shot,
+Following a recent article which corrolated a language's failure with
+the facial hair of its designer, Larry Wall offered this dishearting pic of his
+naked-as-a-baby's butt cheeks. If your resolution is too high to make out the shot,
 then you're missing out on five metric tons of Awesome.
 
-You also might note that Matz' blog is all tricked out with Christmas
-trees and stuff. See, the smartest man alive is still capable of
-appreciating little animated GIFs.
+You also might note that Wall's blog is all tricked out with Christmas
+trees and stuff. 
